packages/prisma/migrations/20241008124232_remove_attribute_value_unique/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- DropIndex
+DROP INDEX "AttributeOption_attributeId_value_key";
```

Comment:
We couldnt unique here - i found out that you couldnt assign the two different member the same value. It has no effect on anything else